web/app/modules/RedisModule.scala

```diff
@@ -8,14 +8,14 @@ import play.api.Configuration
 import play.api.inject.ApplicationLifecycle
 import scala.concurrent.Future
 
-case class RedisConfiguration(host: String, port: Int)
+case class RedisConfiguration(host: String, port: Int, password: String)
 
 /** Provides a RedisClient. */
 @Singleton
 class RedisModule @Inject() (actorSystem: ActorSystem, configuration: Configuration, lifecycle: ApplicationLifecycle) {
   lazy val client: RedisClient = {
-    val config = RedisConfiguration(configuration.get[String]("redis.host"), configuration.get[Int]("redis.port"))
-    val ret = new RedisClient(config.host, config.port)(actorSystem)
+    val config = RedisConfiguration(configuration.get[String]("redis.host"), configuration.get[Int]("redis.port"), password=configuration.get[String]("redis.password"))
+    val ret = new RedisClient(config.host, config.port, password=Option(config.password))(actorSystem)
```

Comment:
er ... this compiles?

Reply:
Oh, I see. Better question: does this work if you run `./dev`?

Reply:
Ah, https://github.com/antirez/redis/issues/3676 -- empty password means no auth.

Reply:
Yep this compiles on my end. Are you getting an error? I haven't tried `./dev`. I ran those `./sbt` and docker/build commands you gave me, built `overview-web` image, pushed to our GCP project and using it in the Kube deployment.